.mergify.yml

```diff
@@ -0,0 +1,39 @@
+---
+pull_request_rules:
+  - name: automatic merge
+    conditions:
+      - and: &base_checks
+        - base=master
+        - -label~=^acceptance-tests-needed|not-ready
+        # wait explicitly for one of the final checks to show up to be on the
+        # safe side even in case of checks not reporting back at all
+        - "check-success~=🐪 Perl"
+        # we don't need to spell out all tests but we can check a minimum
+        # amount
+        - "#check-success=>6"
+        - "#check-failure=0"
+        - "#check-pending=0"
+        - linear-history
+      - and:
+        - "#approved-reviews-by>=2"
+        - "#changes-requested-reviews-by=0"
+        # https://doc.mergify.io/examples.html#require-all-requested-reviews-to-be-approved
+        - "#review-requested=0"
+    actions: &merge
+      merge:
+        method: merge
+  - name: automatic merge on special label
+    conditions:
+      - and: *base_checks
+      - and:
+        # mergify config checks needs at least two rules in "and" so we repeat
+        # one from the base checks
+        - base=master
+        - "label=merge-fast"
```

Comment:
Is this really needed here? I mean for now fast merge makes sense, but in the future shuld be removed right?

Reply:
Well, the difference to the first rule is merely that we don't want to await approval from multiple persons. So for example for a urgent security fix or bugfix that you create I would set that label and as soon as all other checks have passed then mergify would merge regardless of the number of approvals. But I can give that label while tests are still running. mergify will still await successful tests before merging.